main.js

```diff
@@ -32,13 +23,12 @@ var startNewActivityForm = document.querySelector('.start-new-activity')
 var createNewActivityButton = document.querySelector('.create-new-activity-button')
 var userInputs = document.querySelectorAll('input')
 
-
-window.addEventListener('load', displayLocalStorage)
-
+window.addEventListener('load', displayLocalStorage);
 logButton.addEventListener("click", displayCard);
 startActivityButton.addEventListener("click", startActivity);
-createNewActivityButton.addEventListener("click", window.location.reload)
-
+createNewActivityButton.addEventListener("click", function() {
+  window.location.reload()
+})
```

Comment:
Works perfect and gets rid of the 'illegal invocation" console error we were getting.